mappings/net/minecraft/advancement/AdvancementCriterion.mapping

```diff
@@ -1,6 +1,10 @@
 CLASS net/minecraft/class_175 net/minecraft/advancement/AdvancementCriterion
 	FIELD comp_1924 conditions Lnet/minecraft/class_184;
 	METHOD comp_1924 conditions ()Lnet/minecraft/class_184;
+	METHOD method_53698 fromJson (Lcom/google/gson/JsonObject;Lnet/minecraft/class_5257;Lnet/minecraft/class_179;)Lnet/minecraft/class_175;
+		ARG 0 json
+		ARG 1 predicateDeserializer
+		ARG 2 trigger
```

Comment:
```suggestion
		ARG 2 criterion
```

Reply:
The field is called "trigger", plus supplying criterion to deserialize a criterion sounds wrong (yes, yarnbug moment.)